suite/sanity/shrink.go

```diff
@@ -0,0 +1,47 @@
+package sanity
```

Comment:
Missing license header.

Reply:
Including a license header is not currently a standard in the robotest code base.  I've filed https://github.com/gravitational/robotest/issues/231 as a followup work item to add it to this (and all other) file(s).